ui/component/viewers/videoViewer/view.jsx

```diff
@@ -386,15 +386,13 @@ function VideoViewer(props: Props) {
     player.on('loadedmetadata', () => restorePlaybackRate(player));
 
     // Override "auto" to use non-vhs url when the quality matches.
```

Comment:
`// Override "auto" to use non-vhs url when the quality matches.` -- it's no longer the reason we are override it for.
Either:
- Just remove the comment, or
- To avoid complaints of no documentation, maybe edit it to `// Override the "auto" algorithm to post-process the result`.  The reason for overriding can be changed in the future without having to update this comment again.